v4/source/definitions.yaml

```diff
@@ -1489,6 +1489,53 @@ definitions:
         type: string
         description: Set to "all" to mark the channel unread for any new message, "mention" to mark unread for new mentions only. Defaults to "all".
 
+  PluginManifest:
+    type: object
+    properties:
+      id:
+        type: string
+        description: Globally unique identifier that represents the plugin.
+      name:
+        type: string
+        description: Name to be displayed for the plugin.
+      description:
+        type: string
+        description: Description of what the plugin is and does.
+      version:
+        type: string
+        description: Version number for the plugin.
```

Comment:
Propose `Version number of the plugin.`

^For consistency with Name and Description